packages/gatsby/src/schema/graphql-engine/bundle-webpack.ts

```diff
@@ -166,6 +166,12 @@ export async function createGraphqlEngineBundle(
           test: /\.txt/,
           type: `asset/resource`,
         },
+        {
+          test: /\.(graphqls?|gqls?)$/,
```

Comment:
this is to match `const VALID_EXTENSIONS = ['graphql', 'graphqls', 'gql', 'gqls'];` from `graphql-import-node` package, so it doesn't exactly match instructions for loader ( https://www.npmjs.com/package/graphql-tag#webpack-loading-and-preprocessing )